CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.2.1+wb] - 2020-01-22
```

Comment:
if you changed the branches aren't we going to a format: eucalyptus.3-wb-2.2.1?

Reply:
No because `eucalyptus.3-wb-2.2.1` is not a valid package version pattern (according to PyPI standards), and this will lead to more confusion since the fun-apps package version will not match the openedx-docker version.

Reply:
Ok then maybe it was wrong doing it on openfun/edx-platform :-( for the confusion...

Reply:
We'll see (and fix eventually).